### PR TITLE
Add a workflow to ensure that the license report files are modified

### PR DESCRIPTION
**Pull request by armiol**
This PR adds a GitHub Actions workflow to ensure that the license report files are modified within the PR.

The version has been updated to `2.0.0-SNAPSHOT.59`. License report files were updated as well.

In future changesets, the workflow file will be added to `config`.

### CODE REVIEW

File: .github/workflows/ensure-license-reports-modified.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # Configure the checkout of all branches, so that it is possible to run the comparison.
+          fetch-depth: 0
+
+      - name: Check that `pom.xml` and `license-report.md` are modified
+        shell: bash
+        run: chmod +x ./scripts/ensure-required-files-modified-in-PR.sh && ./scripts/ensure-required-files-modified-in-PR.sh
```
Comment (alexander-yevsyukov) [inline .github/workflows/ensure-license-reports-modified.yml:22]: Can we name the file after Corbin Dallas? :) This one is a bit long.

File: scripts/ensure-required-files-modified-in-PR.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	   exit 1;
+	else
+		echo "Detected the modifications in '$1'."
+	fi
+}
+
+echo "Starting to check if all required files were modified within this PR..."
+echo "Comparing \"remotes/origin/$GITHUB_HEAD_REF\" branch to \"remotes/origin/$GITHUB_BASE_REF\" contents."
+
+
```
Comment (alexander-yevsyukov) [inline scripts/ensure-required-files-modified-in-PR.sh:55]: Please remove the extra empty line.

**Review by alexander-yevsyukov — CHANGES_REQUESTED** [review]
Please see my comments. Why don't you make it right in `config`? I'll be able to apply it in `core-java` right away.

**Comment by armiol** [comment]
Closing in favour of [config#254](https://github.com/SpineEventEngine/config/pull/254).

**Pull request closed without merging**